### ISSUE TITLE
Version Managment Project Menu Option??

### ISSUE THREAD

**Issue by ijbranch** [cnpack/cnwizards#206]
Delphi 12.1.
Did/does cnWizards put a Version management menu option under the IDE Projects Menu, at the bottom?
I used to have it there, very convenient, but it has disappeared,
I am unsure which of my plug-ins put it there originally so I am just asking.... :-)

Regards & TIA,
Ian

**Comment by cnpack** [comment]
No, we only make a big 'CnPack' menu in IDE main menubar. No additional entries added to other menus.
Maybe another unknown Delphi plugin?

**Comment by ijbranch** [comment]
Tks for the quick respons.  OK.  It wasn't you, and it wasn't MMX, it isn't GExperts.
Hmm.  Not sure now.  :-(

Regards,
Ian

**Comment by ijbranch** [comment]
Found it!  It was DDevExtensions by Andreas Hausladen.
